test/infamy/netconf.py

```diff
@@ -285,16 +257,28 @@ def get_dict(self, xpath):
         return data.print_dict()
 
     def get_data(self, xpath=None, parse=True):
-        """RPC <get-data> to fetch operational data"""
+        if xpath:
+            xmlns = " ".join([f"xmlns:{m['name']}=\"{m['namespace']}\""
+                                for m in self._modules_in_xpath(xpath)])
+            xpath_filter = f'<xpath-filter {xmlns}>{xpath}</xpath-filter>'
+        else:
+            xpath_filter = None
 
-        if parse is False:
-            return self._get_data(xpath, parse)
+        response = self.ncc.get_data(datastore="ds:operational", filter=xpath_filter)
 
-        data = self._get_data(xpath)
-        if not data:
+        if not response:
             return None
 
-        return data.print_dict()
+        if not parse:
+            return response.raw_reply
+        
+        if len(response.data_ele) == 0:
+            return None
+        
```

Comment:
Just noticed that there are extra whitespace on the two blank lines surrounding this if-statement.  Another thing pylint will complain about.

Reply:
removed the extra whitespace